src/properties-panel/PropertiesPanel.js

```diff
@@ -28,7 +28,8 @@ function renderGroup(props) {
 /**
  * @typedef { {
  *    component: import('preact').ComponentChild,
- *    id: String
+ *    id: String,
+ *    isDirty?: Function
```

Comment:
in the PR we use the term `dirty` to indicate that content exists.

However, from my understanding `dirty` would typically refer to a state where content exists, which is not persisted or synchronzied (for example see our dirty diagram state).

As alternative we could go for `containsContent`, or just turn it around and go for `empty`. Wdyt?

Reply:
> persisted or synchronzied

That's true we use this term differently in other places. Maybe a better term would be `isEdited`, like we have for the title attribute.

Reply:
I changed the term `dirty` to `edited`.